dev/reproducibility.py

```diff
@@ -0,0 +1,95 @@
+import argparse
+import json
+import os
+import shutil
+import nibabel as nib
+
+import numpy as np
+import pandas as pd
+
+from ivadomed import main as ivado
+
+
+def get_parser():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("-c", "--config", required=True, nargs="+", help="Base config file path.")
+    parser.add_argument("-n", "--iterations", default=10, type=int, help="Number of Monte Carlo iterations.")
+    parser.add_argument("-o", "--output-path", nargs="+", dest="output_path", required=True,
+                        type=str, help="Output directory name without extention to save final csv file. There should "
+                                       "be the same number of output files parameters as the number of config files.")
+    return parser
+
+
+def get_results(context):
+    context["command"] = "eval"
+    pred_mask_path = os.path.join(context["log_directory"], "pred_masks")
+    if os.path.exists(pred_mask_path):
+        shutil.rmtree(pred_mask_path)
+
+    # RandomAffine will be applied during testing
+    if "dataset_type" in context["transformation"]["RandomAffine"]:
+        del context["transformation"]["RandomAffine"]["dataset_type"]
+    return ivado.run_command(context)
+
+
+def compute_csa(config, df_results):
+    subject_list = list(df_results.index)
+    df_results = df_results.assign(csa_pred="", csa_gt="", absolute_csa_diff="", relative_csa_diff="")
+    for subject in subject_list:
+        # Get GT csa
+        gt_path = os.path.join(config["loader_parameters"]["bids_path"], "derivatives", "labels", subject.split("_")[0],
+                               "anat", subject + config["loader_parameters"]["target_suffix"][0] + ".nii.gz")
```

Comment:
Here.. that's a bit tricky because in our case we do not want to compute the diff with the GT_down.. but with the GT_raw, which is the best GT we have for the CSA.

Reply:
ahh yes absolutely! I'll add a param for bids path

Reply:
Updated here: 8361dcd